src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Same here, the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), So please edit all action types to be something like this:
`const ADD_BOOK = bookstore/books/ADD_BOOK`

Reply:
@sumairq **Very kind of you to explain the missing pattern. Many thanks**